### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sanjeev9160**
This is a description This is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a descriptionThis is a description

@sushmita-m @madhavan-v @aasiddiq @gauravsaralMs @ashokirla @ashokirla 

image

### CODE REVIEW

**Comment by sanjeev9160** [comment]
@gauravsaralMs Please review the PR... :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 more change
 push to main branch
+eqfef
```
Comment (sushmita-m) [inline README.md:3]: testing review comments  @sanjeev9160

**Comment by gauravsaralMs** [comment]
@sanjeev9160 can you mention me again, i was not mentioned :O

**Comment by sanjeev9160** [comment]
@aasiddiq @gauravsaralMs @sushmita-m @shruti-shukla @shruti-shukla enjoy...

**Comment by sushmita-m** [comment]
@sanjeev9160  which workspace are you signed in last?

File: README.md (hunk 1 above)
Comment (sanjeev9160) [reply README.md:3]: @sushmita-m I will need 10$ for it.

**Comment by gauravsaralMs** [comment]
I am in chatops-app-test-woekspace

@sanjeev9160 @sushmita-m 

https://chatopsapptes-v5a3631.slack.com/archives/C01F3LA2W3U

**Comment by sushmita-m** [comment]
@sanjeev9160  take my swag money :P

**Comment by sushmita-m** [comment]
@sushmita-m cause I just signed in a new workspace.

**Comment by sushmita-m** [comment]
oops forgot to subscribe to comments, test2 `@sushmita-m`

**Comment by sushmita-m** [comment]
test3 `@sushmita-m`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 more change
 push to main branch
```
Comment (sanjeev9160) [inline README.md:2]: Please review @sushmita-m @madhavan-v @gauravsaralMs  @aasiddiq @aasiddiq @shruti-shukla 
@sanjeev9160

**Comment by sanjeev9160** [comment]
comment by @sanjeev9160

**Comment by sushmita-m** [comment]
@sanjeev9160  mention me once

**Comment by sushmita-m** [comment]
@sushmita-m

**Comment by sanjeev9160** [comment]
@sushmita-m mentioned.

**Comment by ashokirla** [comment]
@ashokirla @sanjeev9160 @sushmita-m

**Comment by sushmita-m** [comment]
@sushmita-m

File: README.md (hunk 1 above)
Comment (aasiddiq) [inline README.md:3]: new review comment @sanjeev9160 @sushmita-m @shruti-shukla

**Comment by sushmita-m** [comment]
@sushmita-m

**Comment by sanjeev9160** [comment]
closing it

**Pull request closed without merging**